packages/harpguru-core/src/components/harp-cell/hooks/use-styles/use-styles.ts

```diff
@@ -32,7 +32,7 @@ export const useStyles = (yxCoord: YXCoord): HarpCellStyles => {
       elevation: isActive ? elevation : 0,
       borderRadius,
       borderWidth: thisDegreeId ? borderWidth : 0,
-      borderColor: isActive ? cellColor : borderColor,
+      borderColor: isActive ? 'black' : borderColor,
```

Comment:
`inactiveCellBorderColor` or something could be useful now.

Reply:
The name nominated in the comment above is clearly wrong. Change made on 2e4aed5 if you need to see why.